### DISCUSSION TITLE
Units and their precision

### DISCUSSION THREAD

**Discussion by palorenz** [mr-manuel/venus-os_dbus-mqtt-battery#34] [General]
Hello again,

I have another question regarding all your victron MQTT services:

What are the units and their precision? Are they always the same?

Maybe we could provide json schemas for the MQTT interfaces? I could help with that if I had the necessary information. 

best regards
Patrick

**Comment by mr-manuel** [comment]
You can get the precision from the full JSON example. If you can deliver more precise values, but Venus OS will not have any advantage of that or display them.

**Reply by palorenz** [reply]
Yeah, true for this repo, but not for the grid, or pv MQTT service. But I assume that power, voltage, etc are similar.
